packages/fxa-payments-server/src/store/sequences.test.ts

```diff
@@ -0,0 +1,60 @@
+import { MOCK_PLANS } from '../lib/test-utils';
+
+jest.mock('./actions', () => ({
+  ...jest.requireActual('./actions'),
+  actions: {
+    ...jest.requireActual('./actions').actions,
+    updateSubscriptionPlan: jest
+      .fn()
+      .mockReturnValue({ type: 'updateSubscriptionPlan' }),
+    fetchCustomer: jest.fn().mockReturnValue({ type: 'fetchCustomer' }),
+    fetchSubscriptions: jest
+      .fn()
+      .mockReturnValue({ type: 'fetchSubscriptions' }),
+  },
+}));
+
+import { actions } from './actions';
+import * as thunks from './sequences';
```

Comment:
Why `thunks` here?

Reply:
Hmm, that needs fixing - I think that's a spot where VSCode search & replace & file rename let me down